### PR TITLE
Documentation for dynamic volume limits based on node type

### PR DESCRIPTION
**Pull request by gnufied**
Documentation for dynamic volume limits based on node type.

xref : https://github.com/kubernetes/community/pull/2051

### CODE REVIEW

**Comment by gnufied** [comment]
/sig storage

cc @saad-ali @childsb @jsafrane

**Comment by mistyhacks** [comment]
I rebased to resolve merge conflicts.

**Comment by mistyhacks** [comment]
/assign @steveperry-53 

Thanks!

**Comment by steveperry-53** [comment]
@gnufied @mistyhacks @tengqm See [this commit](https://github.com/kubernetes/website/pull/8871/commits/19820f2bdf9b74d1afaa5311cbfe135a41352e2f) for suggested edits.

Preview: https://deploy-preview-8871--kubernetes-io-vnext-staging.netlify.com/docs/concepts/storage/storage-limits/

**Comment by gnufied** [comment]
@steveperry-53 thank you. will take a look.

@msau42 @ravisantoshgudimetla  Should we  document Balanced volume feature gate somewhere in this document for 1.11 release?

**Comment by msau42** [comment]
Documenting the balanced volumes feature sounds fine.

In general, do we need a new page for this? Or can we append it to the volumes page?

**Comment by ravisantoshgudimetla** [comment]
> Should we document Balanced volume feature gate somewhere in this document for 1.11 release?

I think we should and yes it deserves its own page.

File: content/en/docs/concepts/storage/storage-limits.md [hunk 1]
```diff
@@ -4,61 +4,70 @@ reviewers:
...
+Kubernetes 1.11 introduces dynamic volume limits based on node type. This
+is an alpha feature that supports these services:
 
-Currently this feature is only supported for following volume types:
+- Amazon EBS
+- Google Persistent Disk
 
-- AWS Elastic Block store
-- GCE Persistent Disks
+To enable dynamic volume limits, use the `AttachVolumeLimit` flag.
```
Comment (tengqm) [inline content/en/docs/concepts/storage/storage-limits.md]: What kind of a flag is this? A field in kubelet configuration file or a feature gate?

File: content/en/docs/concepts/storage/storage-limits.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+{{< feature-state state="alpha" >}}
+
+Kubernetes 1.11 introduces dynamic volume limits based on node type. This
+is an alpha feature that supports these services:
+
+- Amazon EBS
+- Google Persistent Disk
+
+To enable dynamic volume limits, use the `AttachVolumeLimit` flag.
```
Comment (tengqm) [inline content/en/docs/concepts/storage/storage-limits.md]: Is `AttachVolumeLimit` a new feature gate to be documented in the reference?

File: content/en/docs/concepts/storage/storage-limits.md (hunk 1 above)
Comment (gnufied) [reply content/en/docs/concepts/storage/storage-limits.md]: This is a new feature gate.

**Comment by steveperry-53** [comment]
@gnufied ping

**Comment by zparnold** [comment]
Hello there @gnufied, do you have an update for this? We are past the docs deadline for 1.11 release and need to get this immediately.

**Comment by gnufied** [comment]
@steveperry-53 @zparnold wait, what are we waiting for from me? Are we waiting for documentation of "Balanced volume" feature?  I did not implement that feature. that will be a new feature introduced in 1.11 but I just pointed out that - we do not seem to have documentation for it. @ravisantoshgudimetla should ideally be adding docs for that feature.

**Comment by zparnold** [comment]
Ah, I see. Apologies. I picked you out of the list because you opened the PR. I'm scanning my spreadsheet and trying to see what I can get merged. 😄

**Comment by ravisantoshgudimetla** [comment]
> @ravisantoshgudimetla should ideally be adding docs for that feature.

That is right. I missed the notification since it was not part of features... Just to be clear, @gnufied, are we fine with creating this document in other page?If yes, where?

**Comment by steveperry-53** [comment]
I think we should get this merged for the 1.11 release.

But soon we should get @tengqm's question answered:
> Is AttachVolumeLimit a new feature gate to be documented in the reference?

And when we say, "use the `AttachVolumeLimit` flag", we should say on which component. kube-apiserver? Other comonent(s)?

Holding for approval by @mistyhacks.
/hold
It's my understanding that @mistyhacks is out today (Monday), but should be able to approve this in Tuesday.

/lgtm

**Pull request merged**